how-to/customize-workspace/client/src/modules/auth/example/util.ts

```diff
@@ -0,0 +1,24 @@
+import type { ExampleUser } from "./shapes";
+
+export function getCurrentUser(userSessionId: string): ExampleUser {
```

Comment:
Why use the `userSessionId` as the key for localstorage, this could potentially end up with lots of debris for old sessions not cleared correctly. Wouldn't it be better to have a fixed key of e.g. `current-user`, I know in this example it would be fixed because the sessionId is from config, but there can only every be one session

Reply:
dropped session key